urban-os/values.yaml

```diff
@@ -79,3 +88,282 @@ kubernetes-data-platform:
+        datasource: Prometheus
+      urban-os-overview:
+        gnetId: 14806
+        revision: 1
+        datasource: Prometheus
+      urban-os-kube-all-nodes:
+        gnetId: 14810
+        revision: 1
+        datasource: Prometheus
+      urban-os-kafka-exporter:
+        gnetId: 14809
+        revision: 1
+        datasource: Prometheus
+      urban-os-phoenix-api-metrics:
+        gnetId: 14811
+        revision: 1
+        datasource: Prometheus
+    # Additional deployment specific dashboards may be
+    # added under the "default" provider:
+    # default:
+    #   custom-dashboard-name:
+    #     gnetId: 14811 (dashboard id from https://grafana.com/grafana/dashboards)
+    #     revision: 1
+    #     datasource: Prometheus
+
+prometheus:
+  server:
+    resources:
+      limits:
+        memory: 1Gi
+        cpu: 500m
+      requests:
+        memory: 1Gi
+        cpu: 500m
+    service:
+      type: NodePort
+  serverFiles:
+    # Any additional alerts can go here, and will be combined with alerting_rules
+    alerts: {}
+
+    rules:
+      groups:
+        - name: pipeline.rules
+          interval: 30s
+          rules:
+            - expr: label_replace(label_replace(sum(rate(kafka_topic_partition_current_offset[2m]) > 0) by (topic), "dataset_id", "$2", "topic", "(raw|transformed)-(.*)"), "stage", "$1", "topic", "(raw|transformed)-(.*)") * on (dataset_id) group_left(dataset_title, source_type, org_name) dataset_info_gauge
+              record: "pipeline:topic:throughput"
+            - expr: label_replace(label_replace(label_replace(sum(kafka_consumergroup_lag{consumergroup!~"console.*"} > 0) by (consumergroup, topic, dataset), "dataset_id", "$3", "consumergroup", "(.+)-(raw|transformed)-(.*)"), "stage", "$2", "consumergroup", "(.+)-(raw|transformed)-(.*)"), "app", "$1", "consumergroup", "(.+)-(raw|transformed)-(.*)") * on (dataset_id) group_left(dataset_title, source_type, org_name) dataset_info_gauge
+              record: "pipeline:topic:lag"
+            - expr: sum(rate(events_handled_count{event_type!="data:extract:end"}[2m]) > 0) by (app, dataset_id, event_type) * on (dataset_id) group_left(dataset_title, source_type, org_name) dataset_info_gauge
+              record: "pipeline:event_stream:events_handled"
+            - expr: label_replace(kafka_consumergroup_lag{consumergroup=~"(.*event-stream|.*events)"}, "app", "$1", "consumergroup", "(.+)-(event-stream|events)") > 0
+              record: "pipeline:event_stream:lag"
+            - expr: sum(label_replace(rate(fluentd_input_status_num_records_total{tag=~".*(forklift|reaper|voltron|valkyrie|odo|discovery-api).*"}[5m]), "name", "$1", "tag", "kube.var.log.containers.(.+?)-.*")) by (name) > 0
+              record: "pipeline:log_message_rate"
+
+    alerting_rules.yml:
+      groups:
+        # Blackbox probe
+        - name: Sites
+          rules:
+            - alert: SiteDown
+              expr: probe_success{instance!~".*(/api/v1/predictions)"} == 0
```

Comment:
This api endpoint is specific to SCOS, though its only excluded here. We could try removing the exclusion and see whether this is still relevant.

Reply:
See it twice below as well

Reply:
Removed `{instance!~".*(/api/v1/predictions)"}` and noticed that it functions the same as prod does currently, so I feel alright about leaving it removed in the public chart, and not accounting for it in `urban-os-deploy`.

removed + pushed up